### ISSUE TITLE
[a11y]: Missing aria-hidden attribute for SharkFinIcon in UI Shell

### ISSUE THREAD

**Issue by vgowrabathuni** [carbon-design-system/carbon-labs#1184]
### Browser

Chrome

### Operating System

MacOS

### Package version

v0.87.0

### React version

React v18.3.1

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker - Latest Deployment

### Description

Currently we are seeing a11y violations for the SharkFinIcon in the UI Shell.

Violation
The SVG element has no accessible name
A non-decorative SVG element must have an accessible name

### WCAG 2.1 Violation

1.1.1 Non-text Content

### Reproduction/example

https://labs.carbondesignsystem.com/react/iframe.html?globals=&args=&id=components-uishell--demo&viewMode=story

### Steps to reproduce

Navigate to the above mentioned URL and scan the page with Equal Access Accessibility Checker to see the violations

<img width="1715" height="973" alt="Image" src="https://github.com/user-attachments/assets/1b8c6872-63f5-4ae2-843d-21c14f7fee28" />

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-labs/blob/main/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon-labs/issues) for duplicate problems
